core/CliMulti/CliPhp.php

```diff
@@ -49,10 +49,6 @@ public function findPhpBinary()
             $bin = shell_exec('which php');
         }
 
-        if (!$this->isValidPhpType($bin)) {
-            $bin = shell_exec('which php5');
```

Comment:
@Findus23 not sure if it's a thing but could do `which php7` (and maybe `which php8`)?

Reply:
I don't think php7 exists on any (Debian) system, only `php7.4` etc.
Not sure if listing all of them helps (if I understand it correctly this would matter if there was no `php` in PATH, but a `php7.4` which seems quite rare).

Reply:
👍 not needed